index.html

```diff
@@ -2225,6 +2225,28 @@ <h2>The <dfn>record type</dfn> string</h2>
   </div>
   </section>
 
+  <section><h3>Aborting pending scan operation</h3>
+  <div>
+    To attempt to <dfn>abort a pending scan operation</dfn> on an
+    <a>environment settings object</a>, perform the following steps:
+    <ol class=algorithm>
+      <li>
+        If there is no <a>pending read tuple</a> |tuple|, abort these steps.
+      </li>
+      <li>
+        If |tuple|'s reader has already initiated an ongoing NFC data transfer,
```

Comment:
Can we link to the definition of an "ongoing NFC data transfer"?

Reply:
We don't do that for writes, either. How can we define it better?